SCClassLibrary/Common/Streams/History.sc

```diff
@@ -151,6 +158,10 @@ History { 		// adc 2006, Birmingham; rewrite 2007.
 
 	play { |start=0, end, verbose=true|	// line numbers;
 		// starting from past 0 may not work.
+		if (lines.isEmpty) {
+			"History.current is empty, cannot play.".postln;
```

Comment:
this isn't strictly true, right? not sure why this check was added in preference to just doing a no-op.

Reply:
Well, when users play a history (current or recorded) and nothing happens, they may be surprised. Next thing they would check is probably whether this history is empty - so why not tell them? History is used in live coding performances, and anything that helps and speeds up orientation live is desirable I think.
Maybe the line should be  "History:play - the history is empty, so there is nothing play yet.".postln;
?

Reply:
changed it be more accurate, informative, and less worry-inducing: 
```
		if (lines.isEmpty) {
			"% - % is empty, so there is nothing to play.".postf(
				thisMethod,
				if (this == History.current) { "History.current" } { "this history" }
			);
```